sdk/identity/identity/BREAKING_CHANGES.md

```diff
@@ -1,5 +1,9 @@
 # Breaking Changes
 
+## 3.2.0
```

Comment:
We don't have the GA out yet for this version.

Reply:
I think we might still include this under 3.2.0 even though it's not out yet. My reading of these files is "these are the breaking changes included in these GA versions." Even if it isn't released, this was the GA version under which we took this change.